### ISSUE TITLE
IMAT: Jaws requiring power cycling and occasionally failing to home

### ISSUE THREAD

**Issue by DominicOram** [ISISComputingGroup/IBEX#6631]
As an IS on IMAT I would like my jaws to be reliable and moving them to be as easy as possible. Currently there are a number of issues that when taken together are very frustrating:

* The drive cards for the jaws require power cycling every time I do a move
* This causes the encoder and motor steps to go out of sync and the Galil to believe it is at a soft limit when it shouldn't be (e.g. https://github.com/ISISComputingGroup/IBEX/issues/4543)
*  Homing the jaws (via the homing script) is then not always that reliable and so the homing script is very cautious and slow (~20 minutes to home everything). It may also be that homing speeds are slow

## Acceptance Criteria
* The reason that the drive card power cycling is required is determined (and fixed if possible)
* Homing is made more reliable and faster

**Comment by FreddieAkeroyd** [comment]
Need to go down with Jamie to beamline to investigate

**Comment by FreddieAkeroyd** [comment]
May be able to try the new galil driver as part of fixing?

**Comment by DominicOram** [comment]
@rerpha and I went down to IMAT on 09/11. We could not reproduce the underlying issue of cards needing to be power cycled. However, we did find that homing the axes was slow, mainly due to the limitation that homing multiple axes at once causes the homing to stall and fail so the must be homed individually. Next step is to put additional logging in the homing routine to work out where it is stalling
